modules/ossm-about-accessing-bookinfo-application-using-gateway.adoc

```diff
@@ -0,0 +1,8 @@
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-accessing-bookinfo-application-using-gateway"]
+= About accessing the Bookinfo application using a gateway
+:context: ossm-about-accessing-bookinfo-application-using-gateway
+
+The {SMProductName} Operator does not deploy Ingress or Egress Gateways. Gateways are not part of the control plane. As a security best-practice, Ingress and Egress Gateways should be deployed in a different namespace than the namespace that contains the control plane.
```

Comment:
I would probably just use "Gateways" here rather than naming `Ingress` and `Egress` specifically because you can also have "east/west gateways" and just using "Gateways" covers all the use cases.

Reply:
Thank you.

Done.

Reply:
done